locations/spiders/arbys.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+import json
+import scrapy
+from urllib import parse
+from locations.items import GeojsonPointItem
+
+STATES = ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DC", "DE", "FL", "GA",
+          "HI", "ID", "IL", "IN", "IA", "KS", "KY", "LA", "ME", "MD",
+          "MA", "MI", "MN", "MS", "MO", "MT", "NE", "NV", "NH", "NJ",
+          "NM", "NY", "NC", "ND", "OH", "OK", "OR", "PA", "RI", "SC",
+          "SD", "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI", "WY"]
+
+
+class ArbysSpider(scrapy.Spider):
+
+    name = "arby"
+    allowed_domains = ["www.arbys.com"]
+    download_delay = 1.5
```

Comment:
If you spider from the directory of all stores you can probably reduce the download delay to zero or to 0.5 or something.

Reply:
👍